### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getLastCursor' of undefined

### ISSUE THREAD

**Issue by andela-ooranagwa** [atom/status-bar#146]
[Enter steps to reproduce below:]

1. Select preferences from atom menu bar drop down

**Atom Version**: 1.7.3
**System**: Mac OS X 10.11.4
**Thrown From**: [status-bar](https://github.com/atom/status-bar) package, v1.1.2

### Stack Trace

Uncaught TypeError: Cannot read property 'getLastCursor' of undefined

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:1002

TypeError: Cannot read property 'getLastCursor' of undefined
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/status-bar/lib/cursor-position-view.js:60:53
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at TextEditor.module.exports.TextEditor.cursorMoved (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:1732:27)
    at Selection.module.exports.Selection.markerDidChange (/Applications/Atom.app/Contents/Resources/app.asar/src/selection.js:927:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/selection.js:41:24
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at TextEditorMarker.module.exports.TextEditorMarker.notifyObservers (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor-marker.js:264:27)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/text-editor-marker.js:65:26
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at Marker.module.exports.Marker.emitChangeEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/marker.js:396:20)
    at Marker.module.exports.Marker.update (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/marker.js:342:12)
    at MarkerLayer.module.exports.MarkerLayer.restoreFromSnapshot (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/marker-layer.js:256:18)
    at TextBuffer.module.exports.TextBuffer.restoreFromMarkerSnapshot (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1333:79)
    at TextBuffer.module.exports.TextBuffer.revertToCheckpoint (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:879:14)
    at TextBuffer.module.exports.TextBuffer.transact (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:831:14)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/whitespace/lib/whitespace.js:52:25
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at TextBuffer.module.exports.TextBuffer.saveAs (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1091:20)
    at TextBuffer.module.exports.TextBuffer.save (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1083:19)
    at TextEditor.module.exports.TextEditor.save (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:660:26)
    at Pane.module.exports.Pane.saveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:718:18)
    at Object.module.exports.autosavePaneItem (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/autosave/lib/autosave.js:58:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/autosave/lib/autosave.js:23:26
    at atom-pane.PaneElement.activeItemChanged (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-element.js:152:25)
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at Pane.module.exports.Pane.setActiveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:282:22)
    at Pane.module.exports.Pane.activateItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:431:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:533:18
    at process._tickCallback (node.js:392:9)
```

### Commands

```
     -0:46.5.0 tree-view:add-folder (span.name.icon.icon-file-directory)
     -0:42.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:33.7.0 tree-view:add-file (span.name.icon.icon-file-directory)
     -0:30.3.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:27.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:19 tabs:split-right (li.texteditor.tab.sortable.active.right-clicked)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
activate-power-mode, v0.5.2
atom-material-syntax, v0.4.6
atom-material-ui, v1.2.10
autoclose-html, v0.23.0
file-icons, v1.7.8
highlight-selected, v0.11.2
merge-conflicts, v1.4.2

# Dev
No dev packages
```

**Comment by phoenixeliot** [comment]
This appears to occur when the cursor is on a newline at the end of a file that Whitespace automatically trims, and you have autosave enabled.

Repro steps:

- In Whitespace's settings, enable "Ensure Single Trailing Newline"
- In config.cson, set: ```
  autosave:
    enabled: true
```
- Create a new file and save it somewhere
- Hit the enter key a few times to type out extra newlines
- Leave the cursor on the last line (or any of the ones that will be trimmed)
- Hit cmd-, (or open Preferences from the menu)

The error should display.

**Comment by Ben3eeE** [comment]
Was able to reproduce in safe mode with the steps @phoenixeliot posted.

**Atom Version**: 1.7.4
**System**: Microsoft Windows 7 Professional 
**Thrown From**: [status-bar](https://github.com/atom/status-bar) package, v1.1.2


### Stack Trace

Uncaught TypeError: Cannot read property 'getLastCursor' of undefined

```
At C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\src\pane.js:1002

TypeError: Cannot read property 'getLastCursor' of undefined
    at C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\status-bar\lib\cursor-position-view.js:60:53
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
    at TextEditor.module.exports.TextEditor.cursorMoved (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\src\text-editor.js:1732:27)
    at Selection.module.exports.Selection.markerDidChange (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\src\selection.js:927:21)
    at C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\src\selection.js:41:24
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
    at TextEditorMarker.module.exports.TextEditorMarker.notifyObservers (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\src\text-editor-marker.js:264:27)
    at C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\src\text-editor-marker.js:65:26
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
    at Marker.module.exports.Marker.emitChangeEvent (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\text-buffer\lib\marker.js:396:20)
    at C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\text-buffer\lib\marker-layer.js:288:27
    at Set.forEach (native)
    at MarkerLayer.module.exports.MarkerLayer.emitChangeEvents (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\text-buffer\lib\marker-layer.js:284:46)
    at TextBuffer.module.exports.TextBuffer.emitMarkerChangeEvents (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:1344:34)
    at TextBuffer.module.exports.TextBuffer.transact (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:852:12)
    at C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\whitespace\lib\whitespace.js:52:25
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
    at TextBuffer.module.exports.TextBuffer.saveAs (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:1091:20)
    at TextBuffer.module.exports.TextBuffer.save (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:1083:19)
    at TextEditor.module.exports.TextEditor.save (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\src\text-editor.js:660:26)
    at Pane.module.exports.Pane.saveItem (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\src\pane.js:718:18)
    at Object.module.exports.autosavePaneItem (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\autosave\lib\autosave.js:58:21)
    at C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\autosave\lib\autosave.js:23:26
    at atom-pane.PaneElement.activeItemChanged (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\src\pane-element.js:152:25)
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
    at Pane.module.exports.Pane.setActiveItem (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\src\pane.js:282:22)
    at Pane.module.exports.Pane.activateItem (C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\src\pane.js:431:21)
    at C:\Users\lineri\AppData\Local\atom\app-1.7.4\resources\app.asar\src\workspace.js:533:18
```

**Comment by jugglingnutcase** [comment]
i'm also seeing this error, sorry i'm not sure how to reproduce this issue. i was editing a file and then hit <kbd>ctrl+,</kbd> to launch the settings view.

**Atom Version**: 1.9.0
**System**: Microsoft Windows 10 Pro
**Thrown From**: [status-bar](https://github.com/atom/status-bar) package, v1.4.0


### Stack Trace

Uncaught TypeError: Cannot read property 'getLastCursor' of undefined

```
At C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\src\pane.js:1026

TypeError: Cannot read property 'getLastCursor' of undefined
    at C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\status-bar\lib\cursor-position-view.js:62:53
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
    at TextEditor.module.exports.TextEditor.cursorMoved (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\src\text-editor.js:1925:27)
    at Selection.module.exports.Selection.markerDidChange (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\src\selection.js:921:21)
    at C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\src\selection.js:41:24
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
    at DisplayMarker.module.exports.DisplayMarker.notifyObservers (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\text-buffer\lib\display-marker.js:243:27)
    at C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\text-buffer\lib\display-marker.js:52:26
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
    at Marker.module.exports.Marker.emitChangeEvent (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\text-buffer\lib\marker.js:422:20)
    at Marker.module.exports.Marker.update (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\text-buffer\lib\marker.js:363:12)
    at MarkerLayer.module.exports.MarkerLayer.restoreFromSnapshot (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\text-buffer\lib\marker-layer.js:258:18)
    at TextBuffer.module.exports.TextBuffer.restoreFromMarkerSnapshot (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:1354:79)
    at TextBuffer.module.exports.TextBuffer.revertToCheckpoint (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:898:14)
    at TextBuffer.module.exports.TextBuffer.transact (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:850:14)
    at C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\whitespace\lib\whitespace.js:52:25
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
    at TextBuffer.module.exports.TextBuffer.saveAs (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:1113:20)
    at TextBuffer.module.exports.TextBuffer.save (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:1105:19)
    at TextEditor.module.exports.TextEditor.save (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\src\text-editor.js:740:26)
    at Pane.module.exports.Pane.saveItem (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\src\pane.js:734:18)
    at Pane.saveItem (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\src\pane.js:3:61)
    at Object.module.exports.autosavePaneItem (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\autosave\lib\autosave.js:58:21)
    at C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\autosave\lib\autosave.js:23:26
    at atom-pane.PaneElement.activeItemChanged (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\src\pane-element.js:152:25)
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:125:28)
    at Pane.module.exports.Pane.setActiveItem (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\src\pane.js:284:22)
    at Pane.module.exports.Pane.activateItem (C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\src\pane.js:433:21)
    at C:\Users\james_lu69\AppData\Local\atom\app-1.9.0\resources\app.asar\src\workspace.js:535:18
```

### Commands

```
     -0:16.1.0 vim-mode:activate-insert-mode (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:15.5.0 vim-mode:activate-normal-mode (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -0:15.1.0 vim-mode:move-to-line (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:14.7.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:14.5.0 vim-mode:insert-after-end-of-line (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:13.9.0 core:backspace (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -0:10.7.0 vim-mode:activate-normal-mode (atom-text-editor.editor.vim-mode.is-focused.insert-mode.autocomplete-active)
     -0:10.6.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:10.4.0 core:save (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:09.2.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:09.2.0 vim-mode:move-to-line (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:09.2.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:09.2.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
  4x -0:09.2.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:08.4.0 vim-mode:insert-below-with-newline (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:05.4.0 settings-view:open (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
advanced-open-file, v0.16.3 (inactive)
atom-yeoman, v0.3.15 (active)
autocomplete-emojis, v2.5.0 (active)
editorconfig, v1.4.1 (active)
jekyll, v1.4.2 (inactive)
json-schema, v0.1.15 (active)
language-cshtml, v0.2.1 (active)
language-liquid, v0.5.1 (active)
language-powershell, v2.1.1 (active)
linter, v1.11.14 (active)
omnisharp-atom, v0.30.4 (active)
vim-mode, v0.65.1 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.51.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.43.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.3.2 (inactive)
one-light-ui, v1.3.2 (active)
one-dark-syntax, v1.2.0 (inactive)
one-light-syntax, v1.2.0 (active)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.2 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.0 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.0 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.38.1 (active)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
find-and-replace, v0.198.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.64.1 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.238.0 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.0 (active)
styleguide, v0.46.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.98.0 (active)
timecop, v0.33.1 (active)
tree-view, v0.208.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.32.2 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.0 (active)
language-csharp, v0.12.1 (active)
language-css, v0.36.2 (active)
language-gfm, v0.86.0 (active)
language-git, v0.13.0 (active)
language-go, v0.42.0 (active)
language-html, v0.44.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.22.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.0 (active)
language-less, v0.29.3 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.0 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.68.5 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.52.0 (active)
language-shellscript, v0.22.3 (active)
language-source, v0.9.0 (active)
language-sql, v0.21.1 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.8 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by 50Wliu** [comment]
Fixed by #164.
